tesseract/tesseract_planning/tesseract_motion_planners/src/simple/step_generators/lvs_interpolation.cpp

```diff
@@ -33,77 +33,477 @@ TESSERACT_COMMON_IGNORE_WARNINGS_PUSH
 #include <memory>
 TESSERACT_COMMON_IGNORE_WARNINGS_POP
 
+#include <algorithm> // std::max
```

Comment:
This needs to go inside the push/pop macros since it is a system library. Also comment should be removed.